src/client/index.ts

```diff
@@ -3,6 +3,16 @@ import to from 'await-to-js';
 export type User = Record<string, string | number | boolean>;
 export type Token = string | undefined;
 export type JWTPayload = Record<string, string>;
+export type KeyCloakApiTokenPayload = {
+  upgraded: boolean;
+  access_token: Token;
+  expires_in: number;
+  refresh_expires_in: number;
+  refresh_token: Token;
+  token_type: string;
+  'not-before-policy': number;
+};
+export type TokenPayload = JWTPayload | KeyCloakApiTokenPayload;
```

Comment:
Perhaps the `TokenPayload` type doesn't need to be exported outside of this client module. I tried to look in all places (excluding tests) where it's used and it seems that it's actually always `JWTPayload`. There is some code that seems to handle also `KeyCloakApiTokenPayload`, but does that code ever get executed in reality?

Reply:
Merged some changes, so now exports not done.

Reply:
`TokenPayload` is now used in commit 8f0941b5 but the type is only introduced later in commit f04063938. Should introduce the type at the same time (or earlier) as it's used.

Reply:
Ah, yes thx, I didn't notice that. Went with just `JWTPayload` and removed the `TokenPayload` type and `KeycloakTokenPayload`.